lxd/db/generate/db/mapping.go

```diff
@@ -273,6 +273,25 @@ func (f *Field) Column() string {
 	return column
 }
 
+// ScalarTableColumn gets the table and column from the join configuration.
+func (f Field) ScalarTableColumn() (string, string, error) {
+	join := f.Config.Get("join")
```

Comment:
I've seen this pattern quite a bit, shall we add a function that centralizes this rather than duplicating the logic in multiple places?